src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -234,6 +234,8 @@ protected function getDateTimeWithOptionalFormat($format, $value)
     protected function getDateTime($value)
     {
         try {
+            $validFormat = new DateTime($value);
```

Comment:
variable unused?

Reply:
Good point. I had thought I needed to assign a variable to trigger the exception if $value could not be parsed.